tests/install_ubuntu_packages.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/bash
+
+# Add ubuntu repository
+sudo add-apt-repository "deb [arch=amd64] http://archive.ubuntu.com/ubuntu focal main universe"
```

Comment:
Because this line is hardocded to "focal" could you please add a verification?
I would like to be notified when ubuntu-latest will become newer.